### PR TITLE
Sprite fixes

### PR DESCRIPTION
**Pull request by Bod9001**
fixes a few issues

### CODE REVIEW

**Comment by corp-0** [comment]
``SpriteHandlerHere`` sounds odd for a variable name. Why was it changed?

**Comment by Bod9001** [comment]
idk couldn't think of a better name

**Comment by corp-0** [comment]
I think we have a couple lowercase public getters that just repeat the type name, like ``playerHealth`` in PlayerScript class. Not quite sure if that's the best solution, given our style guidelines. I invoke the style police @ReiderJack

**Comment by ReiderJack** [comment]
Yes, please fix

**Comment by Bod9001** [comment]
with what?

File: UnityProject/Assets/Scripts/Consoles/ConsoleScreenAnimator.cs [hunk 1]
```diff
@@ -9,13 +9,31 @@ public class ConsoleScreenAnimator : MonoBehaviour, IAPCPowered
...
+		}
+		set
+		{
+			spriteHandler = value;
+		}
+	}
+
+	[SerializeField]
+	private SpriteHandler spriteHandler;
 	public GameObject screenGlow;
```
Comment (ReiderJack) [inline UnityProject/Assets/Scripts/Consoles/ConsoleScreenAnimator.cs]: `ScreenGlow`

File: UnityProject/Assets/Scripts/Consoles/ConsoleScreenAnimator.cs [hunk 2]
```diff
@@ -9,13 +9,31 @@ public class ConsoleScreenAnimator : MonoBehaviour, IAPCPowered
 
 	private bool isAnimating = false;
 	public float timeBetweenFrames = 0.1f;
```
Comment (ReiderJack) [inline UnityProject/Assets/Scripts/Consoles/ConsoleScreenAnimator.cs]: `TimeBetweenFrames`

File: UnityProject/Assets/Scripts/Sprite Handler/SpriteDataSO.cs [hunk 3]
```diff
@@ -31,8 +31,8 @@ public class Frame
 	public void Awake()
```
Comment (ReiderJack) [inline UnityProject/Assets/Scripts/Sprite Handler/SpriteDataSO.cs]: Why it's `public`?

File: UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs [hunk 4]
```diff
@@ -45,6 +45,7 @@ public class SpriteHandler : MonoBehaviour
 
 	private bool subCatalogueChanged = false;
```
Comment (ReiderJack) [inline UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs]: `isSubCatalogueChanged`?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs]: I feel like this is another case of ``is`` not being a good representation of what the variable represents.

**Comment by Bod9001** [comment]
man, it compiles for me fine I don't get what it's doing

**Comment by ThatDan123** [comment]
Did you try rerunning it?

Have you ran the test running in editor?

Have you tried building it from editor?

If those last two pass then i guess its just a github thing.

**Pull request merged**